integration/test/performancetest/performance_test.go

```diff
@@ -78,8 +77,10 @@ func TestPerformance(t *testing.T) {
 	}
 
 	//run tests
+	//@ASK If we should move tps values to seperate config testCases
 	for _, tps := range tpsVals {
 		t.Run(fmt.Sprintf("TPS run: %d", tps), func(t *testing.T) {
+			os.Setenv("TPS",fmt.Sprintf("%d",tps))
```

Comment:
I wanted to use env vars, instead of passing in to make it easier to build upon. However, if this looks complicated I can change it to a parameter.

Reply:
`GetPerformanceMetrics` passes in tps and numLogs as parameters. This overcomplicates the problem, and is more error prone because it depends on nothing else in the environment/process overriding the variable. This should just be passed in as a parameter.